docs/make_work.jl

```diff
@@ -24,11 +25,47 @@ using Documenter, DocumenterCitations
 # Remove the module prefix
 Base.print(io::IO, b::Base.Docs.Binding) = print(io, b.var)
 
+# This is needed to make Documenter.jl follow symlinks, which it is also
+# supposed to be able to do, since there has been a PR about this since 2017:
+# https://github.com/JuliaDocs/Documenter.jl/pull/552`:
+# The Julia docs have the same issue, see:
+# https://github.com/JuliaLang/julia/blob/master/doc/make.jl#L19
+Base.walkdir(str::String) = Base.walkdir(str; follow_symlinks=true)
```

Comment:
Again I am confused: the code in the Julia repo you link to is about copying files; but the line above is type piracy, to change the default value of `follow_symlinks` from `false` to `true`. So perhaps:
> We monkey-patch Base.walkdir to use `true` as default value for `follow_symlinks` (normally `false` is the default), in order to "trick" the Documenter code into following those symlinks.

Reply:
That makes more sense, thanks.